test/ex_json_schema/validator_test.exs

```diff
@@ -667,10 +667,14 @@ defmodule ExJsonSchema.ValidatorTest do
     assert :ok == validate(%{"format" => "date-time"}, false)
   end
 
+  test "format validation succeeds for datetime with tz offset" do
```

Comment:
```suggestion
  test "format validation succeeds for datetime without tz offset" do
```

Reply:
thanks, not my greatest morning with these mistakes 🤦‍♂️

Reply:
😁 🤷